packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.controller.js

```diff
@@ -19,17 +18,48 @@ export default class PciTrainingJobsSubmitController {
     this.atInternet = atInternet;
   }
 
+  addHttpHeader() {
+    this.httpHeader.push({
+      added: false,
+      model: {},
+    });
+  }
+
+  onHttpHeaderAddBtnClick(index) {
+    this.httpHeader[index].added = true;
+    this.job.volumes.push({
+      region: this.httpHeader[index].model.container.region,
+      container: this.httpHeader[index].model.container.name,
+      mountPath: this.httpHeader[index].model.mountPath,
+      permission: this.httpHeader[index].model.permission,
+    });
+    this.addHttpHeader();
+    this.job.validNextStep = this.validateVolume();
+  }
+
+  onHttpHeaderDeleteBtnClick(index) {
+    this.httpHeader.splice(index, 1);
+    this.job.volumes.splice(index, 1);
+    this.validateVolume();
```

Comment:
uselees here ? The result isn't used in the method, and `valid` property of the `oui-step-form` directly call the method
```suggestion
```